addons/web/static/src/core/utils/events.js

```diff
@@ -0,0 +1,108 @@
+// eventUtils.js
+
+const events = new WeakMap();
+
+const delegateEvent = function (ev) {
```

Comment:
TODO: We will remove this utils methods for event delegation as @qsm-odoo is working on another task for this.